src/server/ns_turn_server.c

```diff
@@ -86,26 +86,28 @@ static inline void log_method(ts_ur_super_session *ss, const char *method, int e
     if (!err_code) {
       if (ss->origin[0]) {
         TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO,
-                      "session %018llu: origin <%s> realm <%s> user <%s>: incoming packet %s processed, success\n",
-                      (unsigned long long)(ss->id), (const char *)(ss->origin), (const char *)(ss->realm_options.name),
-                      (const char *)(ss->username), method);
+                      "session %012llu.%d: origin <%s> realm <%s> user <%s>: incoming packet %s processed, success\n",
+                      (unsigned long long)(ss->id), ss->rsid, (const char *)(ss->origin),
+                      (const char *)(ss->realm_options.name), (const char *)(ss->username), method);
       } else {
-        TURN_LOG_FUNC(
-            TURN_LOG_LEVEL_INFO, "session %018llu: realm <%s> user <%s>: incoming packet %s processed, success\n",
-            (unsigned long long)(ss->id), (const char *)(ss->realm_options.name), (const char *)(ss->username), method);
+        TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO,
+                      "session %012llu.%d: realm <%s> user <%s>: incoming packet %s processed, success\n",
```

Comment:
You replace `%018llu` with `%012llu` in a bunch of different places.

Maybe it would be better to make a #define for this string, so it can be reused globally?

Reply:
I assume, that the original author had a reason for not using a macro. Perhaps to avoid clashes when the format gets changed.